govuk_app_config.gemspec

```diff
@@ -25,7 +25,9 @@ Gem::Specification.new do |spec|
   spec.add_dependency "statsd-ruby", "~> 1.4.0"
   spec.add_dependency "unicorn", ">= 5.4", "< 5.8"
 
-  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_development_dependency "activesupport"
+  spec.add_development_dependency "bundler"
```

Comment:
I wonder if we benefit from defining this as a dependency at all, our other gems don't seem to have it and you need bundler before you can install gems. I'd see if there are any consequences to removing it.